changes/ocd-3622.sql

```diff
@@ -0,0 +1,19 @@
+drop table if exists openchpl.pending_certification_result_optional_standard;
+
+CREATE TABLE openchpl.pending_certification_result_optional_standard (
+	pending_certification_result_optional_standard_id bigserial NOT NULL,
```

Comment:
can you just call this "id"?

Reply:
I could, but then it wouldn't have the consistent naming of the Identity columns that the other pending_certification_result_*_* fields do. Is it reasonable to keep the consistency here for this table, especially as we're hoping to remove the table entirely in the flexible-upload epic?